CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,48 @@
+# Code of Conduct
+
+## Welcome
+
+Welcome to our community! We appreciate and welcome contributions from everyone.
+
+## Conduct
+
+While contributing, please keep in mind the following:
+
+- Hate speech, discrimination, and racism are strictly prohibited.
+- Treat all individuals with respect and kindness.
+- Differences in opinion are welcome, but personal attacks or harassment will not be tolerated.
+- Constructive criticism is encouraged, but always be respectful.
+
+## Enforcement
+
+Violations of this Code of Conduct may result in temporary or permanent expulsion from the community at the discretion of the moderators.
+
+If you witness or experience any behavior that violates this Code of Conduct, please report it to the moderators immediately.
+
+## Creating a Codespace and Making Pull Requests
+
+
+### Creating a Codespace:
```

Comment:
Can you remove this section? I feel this does not belong into a CoC and more in a how-to.